### PR TITLE
BIGTOP-3872: Adjust Ambari metrics dependency for Ambari-2.7.5

### PR DESCRIPTION
**Pull request by guyuqi**
### Description of PR
Adjust Ambari metrics dependency for Ambari-2.7.5 in Mpack deployment
Also extend the timeout of grafana service check.

### How was this patch tested?
1. Successfully Build Bigtop Ambari on centos7/rocky8/Debian-11 and smoke tests were passed:
```
./docker-hadoop.sh --create 3 \
--image bigtop/puppet:trunk-rockylinux-8 --memory 16g \
--repo  file:///bigtop-home/output --disable-gpg-check --enable-local-repo \
--stack ambari,bigtop-utils  \
--smoke-tests ambari
```
```
Successfully started process 'Gradle Test Executor 2'

Gradle Test Executor 2 started executing tests.

> Task :bigtop-tests:smoke-tests:ambari:test

org.apache.bigtop.itest.ambari.TestAmbariSmoke > testBlueprints STANDARD_OUT
    [href:http://localhost:8080/api/v1/blueprints, items:[]]

org.apache.bigtop.itest.ambari.TestAmbariSmoke > testStackNameVersion STANDARD_OUT
    [versions:[[Versions:[stack_version:2.0, stack_name:ODPi], href:http://localhost:8080/api/v1/stacks/ODPi/versions/2.0]], Stacks:[stack_name:ODPi], href:http://localhost:8080/api/v1/stacks/ODPi]

Gradle Test Executor 2 finished executing tests.

> Task :bigtop-tests:smoke-tests:ambari:test
Finished generating test XML results (0.013 secs) into: /bigtop-home/bigtop-tests/smoke-tests/ambari/build/test-results/test
Generating HTML test report...
Finished generating test html results (0.032 secs) into: /bigtop-home/bigtop-tests/smoke-tests/ambari/build/reports/tests/test
Now testing...
:bigtop-tests:smoke-tests:ambari:test (Thread[Execution worker for ':' Thread 7,5,main]) completed. Took 4.428 secs.

BUILD SUCCESSFUL in 1m 35s
30 actionable tasks: 7 executed, 23 up-to-date

```

2. Deploy Hadoop , Flink,  solr , Kafaka, Tez, Zeppelin and Ambari Metrics service based on Ambari-2.7.5 :
```
cd bigtop/provisioner/bigtop-stack-provisioner/docker/centos7
Build environment image: ./build-image.sh
Setup Environment: ./build-containers.sh
Access Ambari webui: localhost:8080
Deploy Bigtop serivces
```

### CODE REVIEW

**Comment by guyuqi** [comment]
>You should add comment explaining why this should be commented out.

In some limited hardware resource host(VMs),  grafana service would fail to start for grafana's 'PID_FILE' hadn't been created but the timeout was expired. I temporarily set 120 timeout there for debug. I'm sorry I should refine it and be given explict +1 before merging that PR.

Refined the patch to extend the timeout of grafana service check.
Thanks for your careful and earnest review and commets. @iwasakims

File: bigtop-packages/src/common/ambari/patch5-refine-AMBARI-25599-for-Hadoop-3.4.diff
Comment (iwasakims) [inline bigtop-packages/src/common/ambari/patch5-refine-AMBARI-25599-for-Hadoop-3.4.diff:68]: While I don't think this is the right direction, I'm going to merge this if you can't proceed the release process of Bigtop 3.2.

**Review by iwasakims — APPROVED** [review]

File: bigtop-packages/src/common/ambari/patch5-refine-AMBARI-25599-for-Hadoop-3.4.diff
Comment (kevinw66) [reply bigtop-packages/src/common/ambari/patch5-refine-AMBARI-25599-for-Hadoop-3.4.diff:68]: Thanks for you review, @iwasakims , after some tests on ambari-metrics, we found there are dependency issues when using tarballs provided by their own community

**Pull request merged**
